src/main/java/user/delete_comment/DeleteCommentResponseModel.java

```diff
@@ -0,0 +1,14 @@
+package user.delete_comment;
+
+public class DeleteCommentResponseModel {
+
+    private final String message;
+
+    public DeleteCommentResponseModel(String message){
+        this.message = message;
```

Comment:
We probably want to return more than just a message here. You might want to return the entire new array list of comments, so it can be displayed directly. If you just return a message, what would we display at the front end?